pkg/controller/application/application_controller.go

```diff
@@ -238,6 +247,10 @@ func (c *Controller) syncApplication(key string) error {
 	if app.Annotations == nil {
 		app.Annotations = map[string]string{}
 	}
+	overrideRB, ok := app.Annotations[shipper.RolloutBlocksOverrideAnnotation]
+	if ok {
+		app.Annotations[shipper.RolloutBlocksOverrideAnnotation] = rolloutblock.NewOverride(overrideRB).String()
```

Comment:
This looks like it's taking a string, converting to some object, just to have it back as a string of the same form (or at least most of the times, I realize it removes empty overrides). Am I missing other uses for this?

Reply:
I think mainly to have it in the same order every time

Reply:
It's a stable annotation stringer, yes. It does de-duplication and sorting inside. The code was full of strings.Split() and strings.Join() + de-duplication code all around. The idea is to move it to an object.

Reply:
Sounds like a comment pointing that out would be in order, then, as it's non obvious and relatively important